### ISSUE TITLE
Example Application

### ISSUE THREAD

**Issue by damianesteban** [skylab-inc/Edge#13]
I know that Edge is very new but it would be great to put together some kind of basic demo application.  After I spend a bit more time with the codebase I would be happy to do this :)

**Comment by cloutiertyler** [comment]
That would be excellent! I plan to update the repo with a Dockerfile and host an image on Docker Hub, so that might help out quite a bit with that. It would also be cool to implement a Heroku app, which was asked about in #7.

**Comment by damianesteban** [comment]
That sounds great.  I should have some time this weekend to dive in.

**Comment by cloutiertyler** [comment]
Likewise!

**Comment by cloutiertyler** [comment]
So I've been working on an example application and I think I need to add in a routing framework like Express so that it's much easier to just spin up a quick web app. Still messing around with the design. I'd like it be obvious to both people familiar with reactive programming and those who are not.

**Comment by damianesteban** [comment]
Yes, I think that without at least a basic routing framework it will be
tough.

## 

Damian Esteban

**Comment by cloutiertyler** [comment]
Working on this currently. I made some much needed improvements to Reflex yesterday. I think that should make the design go more smoothly.

**Comment by cloutiertyler** [comment]
@damianesteban Take a look at the routing example in the README here: https://github.com/SwiftOnEdge/Edge/blob/routing/README.md

It was a little tricky to implement, but I think it turned out pretty nice in the end. It still needs quite a bit of work, but the core idea is there.

**Comment by damianesteban** [comment]
Ok, this week I have time (for real).  I'll let you know how it goes.
